Source/gg2/Objects/Projectiles/Rocket.events/User Event 5.xml

```diff
@@ -43,16 +43,19 @@
         with (Character) {
             if (distance_to_object(other) &lt; other.blastRadius and !(team == other.team and id != other.ownerPlayer.object and place_meeting(x, y+1, Obstacle))){
                 var rdir, vectorfactor;
-                rdir = point_direction(other.x ,other.y, (bbox_left+bbox_right)/2, (bbox_top+bbox_bottom)/2);
+                rdir = point_direction(other.x, other.y, (bbox_left+bbox_right)/2, (bbox_top+bbox_bottom)/2);
                 vectorfactor = point_distance(0, 0, power(sin(degtorad(rdir)), 2), power(cos(degtorad(rdir)), 2));
-                if (1 - distance_to_object(other)/other.blastRadius &lt;= 0.25)
+                var fake_distance;
```

Comment:
Is this what you mean by " Make rockets deal minor direct knockback to who they hit, in addition to splash knockback."? Yes, extremely small indeed. Please add a comment saying what this code is meant to do.